packages/patternfly-4/react-core/src/experimental/components/DataToolbar/DataToolbar.tsx

```diff
@@ -0,0 +1,80 @@
+import * as React from 'react';
+import styles from '@patternfly/react-styles/css/components/DataToolbar/data-toolbar';
+import { css } from '@patternfly/react-styles';
+import { DataToolbarExpandableContent } from './DataToolbarExpandableContent';
+import { DataToolbarContext } from './DataToolbarUtils';
+
+export interface DataToolbarProps extends React.HTMLProps<HTMLDivElement> {
+  /** Classes applied to root element of the Data toolbar */
+  className?: string;
+  /** Content to be rendered as rows in the Data toolbar */
+  children?: React.ReactNode;
+  /** Id of the Data toolbar */
+  id: string;
+}
+
+export interface DataToolbarState {
+  /** Flag indicating the if the expandable content is expanded */
+  isExpanded: boolean;
+}
+
+export class DataToolbar extends React.Component<DataToolbarProps, DataToolbarState> {
+  private expandableContentRef = React.createRef<HTMLDivElement>();
+
+  static defaultProps = {
+
+  };
+
+  constructor(props: DataToolbarProps) {
+    super(props);
+
+    this.state = {
+      isExpanded: false,
+    };
+  }
+
+  toggleIsExpanded = () => {
+    this.setState((prevState) => ({
+      isExpanded: !prevState.isExpanded
+    }));
+  }
+
+  closeExpandableContent = () => {
+    this.setState(() => ({
+      isExpanded: false
+    }));
+  }
+
+  componentDidMount() {
+    window.addEventListener('resize', this.closeExpandableContent);
```

Comment:
@karelhala There's an issue where if the toggle drawer is open and the window is resized when the breakpoint is hit the content will not layout correctly.  We're looking for a solution to make it behave the same way that dropdown does.  Will look at that code to see how that one works to see if there's a better solution.

Reply:
This is really unaesthetic because imagine that you are looking at table with this on mobile, click on the expand button, realise that landscape view might be better for this and once you flip the phone this expanded menu is lost and user has to click on it once again.

If the layout is not rendered correctly on window resize I think that is more of a issue in PF core and we shouldn't try to fix it here.

Reply:
@mcarrano Any thoughts on the behavior with the toggle?

Reply:
This is a tough one to answer because it's not clear what the best approach is. I agree that just collapsing the drawer on resize it not great. Maybe if the drawer is open and a breakpoint is crossed, the content should not reconfigure itself (i.e. suppress the behavior in this case). Is that even possible? @mattnolting any thoughts here?

Reply:
@mcarrano - matt nolting is on PTO until Tuesday.

regarding that being possible, yes I believe it is, but we'd have to use the breakpoint passed in by the user and look up the global size of that breakpoint and make it reconfigure itself based on the global size (hopefully no one will have over written it). I may not have time to figure that out today - but I can try. Would that be an acceptable approach @dlabaj?

Reply:
@mcarrano I think, when using a toggle group, the content should reconfigure at the specified breakpoint if expanded content is expanded.